### PR TITLE
Password Policy: Update frontend facing messages

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

Updates the frontend facing messages.

**Why do we need this feature?**

Updating the message gives a clearer description of what's going on, rather than a generic `Internal server error`

image

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #https://github.com/grafana/identity-access-team/issues/574

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/services/user/password.go [hunk 1]
```diff
@@ -8,8 +8,8 @@ import (
 )
 
 var (
-	ErrPasswordTooShort       = errutil.NewBase(errutil.StatusBadRequest, "password-policy-too-short", errutil.WithPublicMessage("New password is too short"))
-	ErrPasswordPolicyInfringe = errutil.NewBase(errutil.StatusBadRequest, "password-policy-infringe", errutil.WithPublicMessage("New password doesn't comply with the password policy"))
+	ErrPasswordTooShort       = errutil.BadRequest("password.password-policy-too-short", errutil.WithPublicMessage("New password is too short")).Errorf("new password is too short")
```
Comment (kalleep) [inline pkg/services/user/password.go:11]: "ErrorF" should probably be called when returning errors. Then it can be used to wrap other error etc

Comment (linoman) [reply pkg/services/user/password.go]: It has been changed as requested.

**Review by kalleep — APPROVED** [review]
Great!

Other thought I have, is there any indication in the ui of what a "strong" password is?

**Comment by linoman** [comment]
> Other thought I have, is there any indication in the ui of what a "strong" password is?

There will be. In the following days we will include some toggles for this feature in the UI. You can follow the progress [here](https://github.com/grafana/identity-access-team/issues/562).

**Pull request merged**